imagersite/imager_images/templates/library.html

```diff
@@ -0,0 +1,28 @@
+{% extends "base.html" %}
+{% block content %}
+
+<h1>{{ user.username }}'s Albums</h1>
+<div class="container">
+     {% for album in user.albums.all %}
+        <div class="col-lg-3 col-md-4 col-sm-5 col-xs-6">
+        <h4 class='text-center'>{{ album.title }}</h4>
+        <a href='/images/albums/{{ album.id }}'><img style='box-shadow: 10px 15px 15px; width: auto; height: 160px;' class="thumbnail" src="{{ MEDIA_URL }}{{ album.cover.file }}"></a>
+        </div>
+        {% if album.cover.file %}
+            src="{{ MEDIA_URL }}{{ album.cover.file }}">
+        {% else %}
+            src="{{ MEDIA_URL }}seattle.jpg">
+        {% endif %}
```

Comment:
I'm a bit confused by this if-else stuff.  What does it go with?  It looks like the end of an HTML tag started elsewhere, but I can't find the start of the tag.